layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/SiteNavigationNewPageElement.vue

```diff
@@ -0,0 +1,56 @@
+<!--
+Copyright (C) 2023 eXo Platform SAS.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+<template>
+  <div>
+    <span class="font-weight-bold text-start text-color body-2 mt-8">{{ $t('siteNavigation.label.pageTemplate') }}</span>
+    <v-select
+      v-model="pageTemplate"
+      :items="pageTemplates"
+      item-text="label"
+      item-value="value"
+      dense
+      class="caption pt-1 mb-5"
+      outlined />
+    <v-img
+      :src="`/layout-management/images/${pageTemplate}.png`"
```

Comment:
put this value as computed
```suggestion
      :src="`/layout-management/images/page-templates/${pageTemplate}.png`"
```